src/App.js

```diff
@@ -1,12 +1,12 @@
 import logo from './logo.svg';
 import './App.css';
-import workList from './components/workList';
+import WorkList from './components/workList.js'
```

Comment:
You don't *need* this `.js` on the end, I wrote this import by hand and it was instinct :p 

The key here is the capitalized component name.